src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -1,3 +1,101 @@
+<script setup>
+import tippy from 'tippy.js'
+import { computed, onBeforeUnmount, onMounted, ref, toRefs } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import LayerTimeConfig from '@/api/layers/LayerTimeConfig.class'
+import {
+    CURRENT_YEAR_WMTS_TIMESTAMP,
+    YEAR_TO_DESCRIBE_ALL_OR_CURRENT_DATA,
+} from '@/api/layers/LayerTimeConfigEntry.class'
+
+const dispatcher = { dispatcher: 'MenuActiveLayersListItemTimeSelector.vue' }
+
+const props = defineProps({
+    layerIndex: {
+        type: Number,
+        required: true,
+    },
+    layerId: {
+        type: String,
+        required: true,
+    },
+    timeConfig: {
+        type: LayerTimeConfig,
+        required: true,
+    },
+    compact: {
+        type: Boolean,
+        default: false,
+    },
+})
+const { layerIndex, layerId, timeConfig, compact } = toRefs(props)
+
+const store = useStore()
+const i18n = useI18n()
+
+const timeSelectorButton = ref(null)
+const timeSelectorModal = ref(null)
+
+const previewYear = computed(() => store.state.layers.previewYear)
+const hasMultipleTimestamps = computed(() => timeConfig.value.timeEntries.length > 1)
+
+const humanReadableCurrentTimestamp = computed(() =>
+    renderHumanReadableTimestamp(timeConfig.value.currentTimeEntry)
+)
+
+let popover = null
+
+onMounted(() => {
+    if (hasMultipleTimestamps.value) {
+        popover = tippy(timeSelectorButton.value, {
+            theme: 'popover-button light-border',
+            content: timeSelectorModal.value,
+            allowHTML: true,
+            placement: 'right',
+            interactive: true,
+            arrow: true,
+            trigger: 'click',
+        })
```

Comment:
We could change the Tippy composable you made so that it can handle a `ref` as `data-tippy-content`... Just wondering if it would make sense to have that extracted as a tippy for future use with rich content...

Reply:
I also thought to change the tippy composable to use a ref, but it made it more complex and I think it is best to keep it simple to be used for simple use case and use directly the tippy for more special case like here for the timeselector button which is not a tooltip on hover but a tooltip interactive on click.